lightning/src/ln/channelmanager.rs

```diff
@@ -947,6 +947,11 @@ macro_rules! handle_chan_restoration_locked {
 						node_id: counterparty_node_id,
 						msg: announcement_sigs,
 					});
+				} else if $channel_entry.get().is_usable() {
```

Comment:
This is gated by the `if let Some(msg) = $funding_locked {` above.  Does it make sense to always send a channel update on reconnect, just in case we updated our policy while the channel was disconnected?

Reply:
Hmm, good point, I pushed the channel_update sending down into the macro callsites.